packages/core/src/types/IKeyManager.ts

```diff
@@ -81,6 +81,35 @@ export interface IKeyManagerDecryptJWEArgs {
   data: string
 }
 
+/**
+ * Input arguments for {@link IKeyManager.keyManagerSign | keyManagerSign}
+ * @public
+ */
+export interface IKeyManagerSignArgs {
+  /**
+   * Key ID
+   */
+  kid: string
+
+  /**
+   * The algorithm to use for signing.
+   * This must be one of the algorithms supported by the KMS for this key type.
+   */
+  alg?: string
+
+  /**
+   * Data to sign
+   */
+  data: string
+
+  /**
+   * If the data is a "string" then you can specify which encoding is used. Default is "utf-8"
+   */
+  enc?: 'utf-8' | 'base16' | 'base64' | 'hex'
```

Comment:
Since `enc` has a different semantics in the JWE world (i.e. encryption algorithm), I'd prefer to change the name of the property.
```suggestion
  encoding?: 'utf-8' | 'base16' | 'base64' | 'hex'
```

Reply:
done